emailCrawler.java

```diff
@@ -49,16 +41,14 @@ public static void main(String args[]) throws IOException {
 	public static void printEmail(ArrayList<String> urlList) throws IOException {
 		for (String url : urlList) {
 			Document doc = Jsoup.connect(url).get();
-
 			// email format
 			Matcher m = Pattern.compile("[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\\.[a-zA-Z0-9-.]+")
 					.matcher(doc.select("a").toString());
 			while (m.find()) {
 				String email = m.group();
 				if (!emailList.contains(email)) {
```

Comment:
Yes, you're right. Because of features of HashSet, we don't need to check it. It's a logical mistake I made.